js/src/components/ImageSelectComponent.js

```diff
@@ -0,0 +1,103 @@
+import { ImageSelect } from "@yoast/components";
+import { openMedia } from "../helpers/selectMedia";
+
+import { Component } from "@wordpress/element";
+/* eslint-disable */
```

Comment:
I hope this was for working on it only 😅 
```suggestion
```

Reply:
Oops  😳